modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSViews.tsx

```diff
@@ -292,6 +297,16 @@ const FDSViews = ({
 		],
 	};
 
+	const TitleRenderer = ({itemData}: {itemData: FDSViewType}) => {
+		return (
+			<div className="table-list-title">
+				<ClayLink href={getViewURL(itemData).toString()}>
+					{itemData.label}
+				</ClayLink>
+			</div>
+		);
+	};
+
```

Comment:
This is good in principle, to add a custom renderer here. Recently we are making effort to standardize interfaces for renderers. As this this would be a first non-cell renderer component, this is a good opportunity to extend it. Please add to renderer.ts

```ts
export interface InternalRenderer extends Renderer {
	component: React.ComponentType<any>;
	label: string;
	name: string;
	type: 'internal';
}
```
Note that a "renrderer" is not just the component, but also its metadata.

Then, in this file,  it would be something like 

```ts
const titleRenderer: InternalRenderer = {
	component: ...,
	label: Liferay.Language.get('title'),
	name: "title",
	type: 'internal',
}
```
,
```ts
schema: {
		titleRenderer,
	},
```
and in List.js:
```ts
const TitleRendererComponent = titleRenderer.component;
```